values.yaml

```diff
@@ -1,17 +1,24 @@
 app:
   name: quickstart
   dbms: none
+#externalDatabase:
+#  host: openshift.serv.run
+#  port: 32432
+#  databaseName: sampledb
+#  username:
+#  password:
 operator:
   supportOpenshift: true
   env:
     ENTANDO_DOCKER_IMAGE_VERSION_FALLBACK: 6.0.0
     #ENTANDO_DOCKER_REGISTRY_OVERRIDE: docker.io # Remove comment if you want to always use a specific docker registry
     #ENTANDO_DOCKER_IMAGE_ORG_OVERRIDE: entando # Remove the comment if you want to always use a specific docker organization
-    ENTANDO_DEFAULT_ROUTING_SUFFIX: your.domain.suffix.com
+    ENTANDO_DEFAULT_ROUTING_SUFFIX: apps.serv.run
```

Comment:
Is it an inattention or is it right as is?

Reply:
you're right let me fix it